packages/react-linked-input/LinkedInput.js

```diff
@@ -10,7 +10,7 @@
 'use strict';
 
 var React = require('react');
-var LinkedValueUtils = require('react/lib/LinkedValueUtils');
+var LinkedValueUtils = require('react-dom/lib/LinkedValueUtils');
```

Comment:
I guess that means that this is a breaking change by any standard then. If we're not planning on updating these.

Also curious about `clone-with-props` et al. that are out there requiring deeply.